docs/pages/getting-started/docker-compose.mdx

```diff
@@ -0,0 +1,217 @@
+---
+title: Getting started with Teleport using Docker Compose
+description: How to get started with Teleport Open Source Edition using Docker Compose locally.
+h1: Get started with Docker Compose
+---
+
+This section will cover:
+
+- Getting started with a local Teleport cluster using docker compose.
+- Using Teleport with OpenSSH, ansible and Teleport's native client, `tsh`.
+
+## Prerequisites
+
+- Teleport v(=teleport.version=) Open Source or Enterprise.
+- Docker v(=docker.version=) or later and [`docker-compose`](https://docs.docker.com/compose/install/) v(=docker.compose.version=) or later.
+
+```bash
+docker-compose version
+# docker-compose version (=docker.compose.version=), build unknown
+
+docker version
+#Client: Docker Engine - Community
+# Version:           (=docker.version=)
+```
+
+## Step 1/3. Start teleport demo lab
+
+Let's use `docker-compose` to start Teleport demo lab - a configured local cluster:
+
+```bash
+# Download the quick start file from our Github repo
+curl -Lso teleport-lab.yml https://raw.githubusercontent.com/gravitational/teleport/v(=teleport.version=)/docker/teleport-lab.yml
+# Start teleport demo lab using docker-compose
+docker-compose -f teleport-lab.yml up
```

Comment:
Should do this in the background too really otherwise it's hard to detach from the console (if you ^Z you get log spam all over the place)

```suggestion
docker-compose -f teleport-lab.yaml up -d
```